libraries/bezier-rs/src/lib.rs

```diff
@@ -1,1447 +1,9 @@
 //! Bezier-rs: A Bezier Math Library for Rust
 
+mod bezier;
```

Comment:
What do you think about calling this module "segment" instead of "bezier"?

Reply:
We decided to defer this change because it would require renaming the `Bezier` and `BezierHandles` structs and those would necessitate lots of comment changes also. This should probably be done before publishing the library though.